.github/CHANGELOG.md

```diff
@@ -12,7 +12,8 @@
     code,distance,ec,boundaries,noise,delta,p_swap,decoder,weight_opts,errors,trials,current_time,simulation_time,mpi_size
     SurfaceCode,3,primal,open,CVLayer,0.09,0.25,MWPM,{'method': 'blueprint', 'integer': False, 'multiplier': 1, 'delta': 0.09},10,100,00:15:50,0.370795,1
     ```
-* Added Plotly backend for visualizing graph states [#103](https://github.com/XanaduAI/flamingpy/pull/103)
+* Added Plotly backend for visualizing graph states. [#103](https://github.com/XanaduAI/flamingpy/pull/103)
+* Users can install FlamingPy from git with pip. [#118](https://github.com/XanaduAI/flamingpy/pull/118)
```

Comment:
This explanation needs some improvement but can be done in the upcoming Special Release PR.